docs/layouts/partials/head.html

```diff
@@ -0,0 +1,41 @@
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+{{- if eq .Title "" }}
+<title>{{ .Site.Title }}</title>
+{{- else }}
+<title>{{ .Title }} - {{ .Site.Title }}</title>
+{{- end }}
+{{- if ne .Description "" }}
+<meta name="description" content="{{ .Description }}">
+{{- else if isset .Site.Params "description" }}
+<meta name="description" content="{{ .Site.Params.description }}">
+{{- end }}
+{{ hugo.Generator }}
+{{- range $i := .Site.Params.faviconSizes }} 
+{{- $favicon := resources.Get "img/favicon.png" }} 
+{{- $image := $favicon.Resize (printf "%dx%d" $i $i) }}
+<link rel="icon" type="image/png" sizes="{{$i}}x{{$i}}" href="{{ $image.Permalink }}"/>
+{{- end }} 
+{{- with resources.Get "img/favicon.png" }} 
+{{- $image := .Resize "180x180" }}
+<link rel="apple-touch-icon" sizes="180x180" href="{{ $image.Permalink }}" />
+{{- end }}
+<link href="{{ .Site.BaseURL }}/index.xml" rel="alternate" type="application/rss+xml">
+<link rel="canonical" href="{{ .Permalink }}">
+<link rel="stylesheet" href="{{"css/theme.min.css" | absURL}}">
+{{ $style := resources.Get "scss/brigade.scss" | toCSS | minify | fingerprint }} 
+<link rel="stylesheet" href="{{ $style.RelPermalink }}">
+<script src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+{{ partial "meta/chroma.html" . -}}
+<script src="https://cdn.jsdelivr.net/npm/jquery@3.4.1/dist/jquery.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/jquery.easing@1.4.1/jquery.easing.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.6/dist/clipboard.min.js"></script>
+<script src="{{ "js/bundle.js" | absURL }}"></script>
+{{- partial "meta/google-analytics-async.html" . -}}
+{{- partial "meta/tag-manager.html" . -}}
+{{- partial "meta/google-site-verification.html" . -}}
+{{- partial "custom-css.html" . -}}
+{{- template "_internal/opengraph.html" . -}}
+{{- template "_internal/twitter_cards.html" . -}}
+{{- template "_internal/schema.html" . -}}
```

Comment:
This file is _copied_ from the theme and minimally modified to include include our favicon and custom stylesheet. Hugo favors this file over the original in `themes/...`